include/cutlass/gemm/kernel/xe_gemm.hpp

```diff
@@ -262,7 +262,7 @@ class GemmUniversal<
     CollectiveEpilogue epilogue{params.epilogue, shared_storage.epilogue};
     epilogue(
       problem_shape_MNKL,
-      subgroup_shape,
+      subgroup_shape, // TODO(joe): Inconsistency here w/ blk_coord_mnkl
```

Comment:
@aacostadiaz the subgroup shape is being passed in here instead of the `WorkgroupTileShape`. However, the coordinate `blk_coord_mnkl` is the work-group coordinate. This doesn't matter for this PR because I'm computing things inside `xe_epilogue.hpp` but it feels inconsistent. Potentially something to address as part of your ongoing work on the coordinates?

Reply:
Yes, that makes sense. I can look into that as part of the fix for coordinates.

Reply:
maybe change to `TODO(codeplay): `?

Reply:
Done :+1: